### ISSUE TITLE
Website is unusable on mobile due to scroll container

### ISSUE THREAD

**Issue by jameshfisher** [helm/helm-www#1393]
Using Chrome on Android, scroll is unusable. The page seems to be wrapped in a container that has its own scroll bar.

**Comment by SaberStrat** [comment]
Yes, scrolling gets sticky on iOS too. It’s super annoying to get the site scrollable again.

It‘s been like this ever since I started reading the site 2021.

**Comment by vekien** [comment]
This is still broken………

2025, one year I’m sure it’ll be fixed.

**Comment by scottrigby** [comment]
I believe this is fixed now! As part of the docusaurus migration. It works well on iOS, and should on android mobile as well. Please open a new issue if there are still specific problems by OS/device. Thanks!
